src/proxy/http/router.rs

```diff
@@ -126,22 +113,18 @@ where
             self.config.capacity,
             self.config.max_idle_age,
         );
-        let span = debug_span!("router", name = self.config.proxy_name);
-        let ctx = logging::Section::Proxy.bg(self.config.proxy_name);
-        let cache_daemon = ctx.future(cache_bg);
-        tokio::spawn(cache_daemon);
-
-        Service { inner, span }
+        tokio::spawn(cache_bg.instrument(info_span!("router")));
```

Comment:
Should the background future be 
```suggestion
        tokio::spawn(cache_bg.instrument(info_span!("router_bg")));
```
or something?